data_collection/gazette/spiders/pr_campo_mourao.py

```diff
@@ -0,0 +1,48 @@
+from dateparser import parse
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCampoMouraoSpider(BaseGazetteSpider):
+    TERRITORY_ID = "4104303"
+    name = "pr_campo_mourao"
+    start_urls = ["https://campomourao.atende.net/?pg=diariooficial&pagina=1"]
+
+    def parse(self, response):
+        url_base = response.url[0:56]
+        page = int(response.url[56:])
+
+        list = response.xpath("//div[@class='nova_listagem ']/div[@class='linha']")
+
+        if not list:
+            return
+
+        for row in list:
+            date = row.xpath("//div[@class='info']/div[@class='data']/text()").get()
+            date = parse(date, languages=["pt"]).date()
+
+            edition_type = row.xpath(
+                "//div[@class='info']/div[@class='tipo']/text()"
+            ).get()
+            edition_number = row.xpath(
+                "//div[@class='info']/div[@class='titulo']/text()"
+            ).get()
+            edition_number = edition_number.split(" ")[1]
+
+            code = row.xpath("//button[@data-acao='download']/@data-codigo").get()
+            id = row.xpath("//button[@data-acao='download']/@data-id").get()
+
+            is_extra = True if edition_type == "Extraordinária" else False
+
+            url = f"https://campomourao.atende.net/atende.php?rot=54002&aca=737&processo=download&codigo={code}&hash={id}"
+
+            yield Gazette(
```

Comment:
You should add the `start_date` class attribute and stop yielding items and requests the moment you find a date below the `start_date`. The default value should be the first available date in the website, in this case: 03/02/2012.

This is important to make running this spider in production everyday (only downloading last day's gazette) without making unnecessary requests. You could look into some spiders that have it (like `am_manaus`) to get the idea of what I mean. And in case you have any doubts, just ask and I'll be here to help